uni/lib/view/library/widgets/library_occupation_card.dart

```diff
@@ -22,8 +22,10 @@ class LibraryOccupationCard extends GenericCard {
   String getTitle(BuildContext context) => S.of(context).library_occupation;
 
   @override
-  Future<Object?> onClick(BuildContext context) =>
-      Navigator.pushNamed(context, '/${DrawerItem.navLibrary.title}');
+  Future<Object?> onClick(BuildContext context) => Navigator.pushNamed(
+        context,
+        '/${DrawerItem.navLibraryReservations.title}',
+      );
```

Comment:
Should clicking the card redirect to reservations?

Reply:
Yes, it has been fixed